tests/bluetooth/controller/mock_ctrl/include/hal/ticker_vendor_hal.h

```diff
@@ -3,38 +3,57 @@
  *
  * SPDX-License-Identifier: Apache-2.0
  */
+
 #define HAL_TICKER_CNTR_CLK_FREQ_HZ 32768U
+#define HAL_TICKER_CNTR_CLK_UNIT_FS 30517578125UL
 
 /* Macro defining the minimum counter compare offset */
 #define HAL_TICKER_CNTR_CMP_OFFSET_MIN 3
 
 /* Macro defining the max. counter update latency in ticks */
 #define HAL_TICKER_CNTR_SET_LATENCY 0
 
+/* Macro defines the h/w supported most significant bit */
+#define HAL_TICKER_CNTR_MSBIT 23
+
+/* Macro defining the HW supported counter bits */
+#define HAL_TICKER_CNTR_MASK 0x00FFFFFF
+
 /* Macro to translate microseconds to tick units.
  * NOTE: This returns the floor value.
  */
-#define HAL_TICKER_US_TO_TICKS(x)                                                                  \
-	(((uint32_t)(((uint64_t)(x) * 1000000000UL) / 30517578125UL)) & HAL_TICKER_CNTR_MASK)
+#define HAL_TICKER_US_TO_TICKS(x) \
+	( \
+		((uint32_t)(((uint64_t) (x) * 1000000000UL) / \
```

Comment:
The multiplication of `1000000000UL` to done several places in the controller code. Suggest to introduce a macro for, similar to the `NSEC_PER_MSEC` and similar macros

Reply:
done